src/core_plugins/kibana/public/management/index.js

```diff
@@ -41,6 +42,9 @@ uiModules
     },
 
     link: function ($scope) {
+      if ($route.current.pathParams.section) {
```

Comment:
Rather than introducing conditional logic here, it would be cleaner to do the redirect in the `uiRoutes.when('/management/:section')` block above. It will also match up with the comment you added right above that block. You can use Angular's [`redirectTo`](https://docs.angularjs.org/api/ngRoute/provider/$routeProvider) on the route definition to perform the redirect.